game-app/game-core/src/main/java/games/strategy/engine/framework/LocalPlayers.java

```diff
@@ -30,7 +29,6 @@ public boolean playing(final GamePlayer player) {
 
   private static boolean isGamePlayerWithPlayerId(
       final Player player, final GamePlayer gamePlayer) {
-    return player.getGamePlayer().equals(gamePlayer)
-        && AbstractHumanPlayer.class.isAssignableFrom(player.getClass());
+    return player.getGamePlayer().equals(gamePlayer) && !player.isAi();
```

Comment:
I'm not sure if in a headless environment the code tries to invoke this over the network. If it does I have to re-introduce a dirty check if this class inherits from some abtract class in the type hierarchy

Reply:
Seems safe